x-pack/plugins/transform/public/app/sections/create_transform/components/step_define/step_define_form.tsx

```diff
@@ -288,6 +289,11 @@ export const StepDefineForm: FC<StepDefineFormProps> = React.memo((props) => {
     };
   });
 
+  const rowCountInfoLabel = i18n.translate('xpack.transform.stepDefineForm.rowCountInfoLabel', {
```

Comment:
For react components `FormattedMessage` component is preferred according to Kibana's guidelines

Reply:
Updated in: [#7c3e699](https://github.com/elastic/kibana/pull/196510/commits/314eec15d2beab13cd35216315d630b6f16003b6)